uds/translator/data_record_definitions/conditional.py

```
@@ -221,30 +222,30 @@
-                                         length=8,
-                                         min_occurrences=1,
-                                         max_occurrences=1,
-                                         unit="DIDs")
-                           for record_number in range(REPEATED_DATA_RECORDS_NUMBER))
-"""Collection of `DIDCount` Data Records."""
-
 _DTC_SNAPSHOT_RECORDS_2020 = tuple(item
                                    for snapshot_record in zip(OPTIONAL_DTC_SNAPSHOT_RECORDS_NUMBERS_LIST,
-                                                              _DID_COUNT_RECORDS,
+                                                              DID_COUNT_RECORDS,
                                                               _DID_RECORDS_2020)
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)